services/management/ia/templates_service.go

```diff
@@ -30,18 +33,27 @@ import (
 	saas "github.com/percona-platform/saas/pkg/alert"
 	"github.com/percona/pmm/api/managementpb"
 	iav1beta1 "github.com/percona/pmm/api/managementpb/ia"
+	"github.com/percona/promconfig"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"gopkg.in/reform.v1"
+	"gopkg.in/yaml.v3"
 )
 
 const (
 	builtinTemplatesPath = "/tmp/ia1/*.yml"
 	userTemplatesPath    = "/tmp/ia2/*.yml"
+
+	ruleFileDir = "/tmp/ia1/"
+
+	// TODO remove once we start using real values
+	dummyParamValue = "80"
 )
 
+var paramRegex = regexp.MustCompile(`\[\[.*?\]\]`)
```

Comment:
Please use text/template with custom delimiters: https://github.com/percona-platform/pocs/blob/e7b00850eec37585a4f69bb15b911fcc8f08a4c7/ia/rule.go#L37

Reply:
i didn't know the conversion code was already there, thanks